### PR TITLE
imu supports frameName

### PR DESCRIPTION
**Pull request by leftys**
Imu cannot publish in frames other than bodyName. Thus doesn't work with nested tf like /robot/part1/part2.

### CODE REVIEW

File: gazebo_plugins/src/gazebo_ros_imu.cpp [hunk 1]
```diff
@@ -124,6 +124,13 @@ void GazeboRosIMU::LoadThread()
   else
     this->update_rate_ = this->sdf->GetElement("updateRate")->Get<double>();
 
+  if (!this->sdf->HasElement("frameName"))
+  {
+    ROS_INFO("Laser plugin missing <frameName>, defaults to <bodyName>");
```
Comment (chapulina) [inline gazebo_plugins/src/gazebo_ros_imu.cpp]: `imu` instead of `Laser`?

**Comment by chapulina** [comment]
Thanks for the contribution! 

Do you happen to have a simple world example using the imu plugin? I noticed there is none in the package yet. It would be great if we could add one to `gazebo_ros_pkgs/gazebo_plugins/test/test_worlds`

**Comment by leftys** [comment]
Hi! I am not using gazebo at the time and forgot most of its usage, so I can't add imu example usage. I fixed the 'Laser' typo however.

**Comment by scpeters** [comment]
@ros-pull-request-builder test this please

**Comment by scpeters** [comment]
the CI is clean now after a rebuild (there was a spurious `apt-get` error)

**Comment by scpeters** [comment]
+1

**Comment by j-rivero** [comment]
@chapulina what you think?

**Comment by chapulina** [comment]
The changes look alright, but I don't know how to test this...

**Comment by scpeters** [comment]
There is an example urdf at http://answers.gazebosim.org/question/4717/imu-sensor-plugin-libgazebo_ros_imuso-strange-data/ but I don't know if it works properly. Or we could add one to the pendulum in gazebo_ros_demos

**Comment by chapulina** [comment]
Ok... We can add an example in the future then. +1

**Pull request merged**